packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -52,6 +51,7 @@ export default function createConnector(connectorDesc) {
       static defaultClassNames = Composed.defaultClassNames;
       static propTypes = connectorDesc.propTypes;
       static defaultProps = connectorDesc.defaultProps;
+      static getSearchParameters = connectorDesc.getSearchParameters; //can be removed?
```

Comment:
Indeed I'm wondering if I should mention this. It's not meant to be used without the usage of "findResults".